lib/optimizely/audience.rb

```diff
@@ -24,23 +24,32 @@ module Optimizely
   module Audience
     module_function
 
-    def user_in_experiment?(config, experiment, attributes, logger)
-      # Determine for given experiment if user satisfies the audiences for the experiment.
+    def user_meets_audience_conditions?(config, experiment, attributes, logger, logging_hash = nil, logging_key = nil)
```

Comment:
nit. `does_user_meet_audience_conditions`

Reply:
I can rename it to match with other SDKs but the idiomatic code for ruby is to name methods like these with a question mark. See for example this link with for is_ and has_. https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/PredicateName